### PR TITLE
Branch batch edit

### PR DESCRIPTION
**Pull request by MinkhantEain**
Batch edit is done I think. Feel free to test

### CODE REVIEW

File: src/main/java/seedu/address/logic/commands/BatchEditCommand.java [hunk 1]
```diff
@@ -0,0 +1,116 @@
...
+
+    public static final String MESSAGE_USAGE = COMMAND_WORD
+            + ": Change the specified tag with new tag"
+            + "Parameters:"
+            + PREFIX_TAG + "TAG" + " "
+            + PREFIX_TAG + "TAG\n"
+            + "Example: " + COMMAND_WORD + " "
+            + PREFIX_TAG + "OLDTAG " + PREFIX_TAG + "NEWTAG";
+
+    public static final String MESSAGE_BATCH_EDIT_EACH_PERSON_SUCCESS = "Tag changed from: %1$s\n";
```
Comment (miuna9890) [inline src/main/java/seedu/address/logic/commands/BatchEditCommand.java:33]: Works great just that the result message shows the old tag before editing

Comment (MinkhantEain) [reply src/main/java/seedu/address/logic/commands/BatchEditCommand.java]: i have added change to section.

Comment (miuna9890) [reply src/main/java/seedu/address/logic/commands/BatchEditCommand.java]: Oki can but inputting a wrong tag shows an empty message. Similiar to edit and delete commands etc i think we could have an error message like tag specified does not exist or smth

File: src/main/java/seedu/address/logic/commands/BatchEditCommand.java [hunk 2]
```diff
@@ -0,0 +1,117 @@
...
+/**
+ * Edit all person with specified tag to change the specified tag with new tag.
+ */
+public class BatchEditCommand extends Command {
+    public static final String COMMAND_WORD = "batch-edit";
+
+    public static final String MESSAGE_USAGE = COMMAND_WORD
+            + ": Change the specified tag with new tag"
+            + "Parameters:"
+            + PREFIX_TAG + "TAG" + " "
```
Comment (yyueyii) [inline src/main/java/seedu/address/logic/commands/BatchEditCommand.java:27]: I think the parameters here should be PREFIX_TAG + "OLDTAG " + PREFIX_TAG + "NEWTAG"
And the Example should be a real life example eg. PREFIX_TAG + "Sec1 " + PREFIX_TAG + "Sec 2 will change all tags named Sec1 to Sec 2"

File: src/main/java/seedu/address/logic/commands/BatchEditCommand.java [hunk 3]
```diff
@@ -0,0 +1,117 @@
...
+        model.updateFilteredPersonList(predicate);
+        List<Person> curentShownList = model.getFilteredPersonList();
+        ArrayList<Person> nonObservableList = new ArrayList<>(curentShownList);
+        StringBuilder feedbackToUser = new StringBuilder();
+
+        for (Person person : nonObservableList) {
+            Person updatedPerson = changeTag(person, oldTag, newTag);
+            model.setPerson(person, updatedPerson);
+            feedbackToUser.append(String
+                    .format(MESSAGE_BATCH_EDIT_EACH_PERSON_SUCCESS_FROM, Messages.format(person))
```
Comment (yyueyii) [inline src/main/java/seedu/address/logic/commands/BatchEditCommand.java:73]: iinw Message.format will show the entire toString() of the entries eg "John Doe r/student...." I don't think this fits the previous MESSAGE_BATCH_EDIT_EACH_PERSON_SUCCESS_FROM/TO part. Maybe can make can just replace w oldtag/newtag toString()?

**Review by yyueyii — CHANGES_REQUESTED** [review]
Some minor changes regarding the CommandResults/Messages. Otherwise LGTM!

File: src/main/java/seedu/address/logic/commands/BatchEditCommand.java (hunk 1 above)
Comment (miuna9890) [reply src/main/java/seedu/address/logic/commands/BatchEditCommand.java]: Yupp LGTM now.

**Review by miuna9890 — APPROVED** [review]
other than changes requested by @yyueyii LGTM

**Review by yyueyii — APPROVED** [review]

**Pull request merged**